### DISCUSSION TITLE
Cockpit running on Rpi - display model?

### DISCUSSION THREAD

**Discussion by cooka90** [cockpit-project/cockpit#16061] [Q&A] [answered]
Hi, we have about 80 rpis in a corporate setting. While I really like Cockpit, it would be really handy to be able to see the model of the pi from the dashboard. It would also be beneficial to be able to see if there are available updates from the dashboard too. Is this possible? 

Many thanks

**Comment by marusak** [comment, marked as answer]
Do you mean `Dashboard` component? If so, then this has been obsoleted for a while already. Also it was limited to 20 devices.
Regarding updates, this has been requested before, see for example https://github.com/cockpit-project/cockpit/issues/14570
